overlord/ifacestate/handlers.go

```diff
@@ -655,12 +704,10 @@ func (m *InterfaceManager) doAutoConnect(task *state.Task, _ *tomb.Tomb) error {
 			continue
 		}
 
-		ts, err := ConnectOnInstall(st, chg, task, plug.Snap.Name(), plug.Name, slot.Snap.Name(), slot.Name)
-		if err != nil {
-			task.Logf("cannot auto-connect plug %s to %s: %s", connRef.PlugRef, connRef.SlotRef, err)
-			continue
+		if err := checkConnectConflicts(st, chg, plug.Snap.Name(), slot.Snap.Name(), task, waitingOnUs); err != nil {
```

Comment:
as discussed we probably want to go back to ignore conflicts and errors here, so here we should ignore conns with conflicts (but probably report other kinds of errors)